packages/manager/modules/vps/src/veeam/vps-veeam.controller.js

```diff
@@ -105,4 +109,20 @@ export default class {
         this.veeamTab.loading = false;
       });
   }
+
+  isValidToSchedule() {
+    return (
+      this.isValidVpsVersionForAutomateBackup() && this.hasAutomatedBackupOption
+    );
+  }
+
+  /**
+   * Used to validate the targeted VPS version
+   * If new version has been raised and it's allow customers to used schedule backup
+   * So add this version into the array.
+   * @returns {boolean}: true is is valid version, otherwise false
+   */
+  isValidVpsVersionForAutomateBackup() {
+    return [VPS.VERSIONS.V1_2019].includes(this.vpsDetails.model.version);
```

Comment:
Can't this be returned by some capability API ?

Reply:
2API maybe then ?

Reply:
Yes it should.
@marie-j is it fast to implemment the purpose into 2API side ?

Reply:
Thanks for suggestion @marie-j.
I implemented it on 2API side.
Completed